README.md

```diff
@@ -42,6 +42,7 @@ default-sort-orders   : object, keys are fields, values are 'A' or 'D', to
                         indicate the order to sort by on first click
 csv-exportable        : boolean, if true, show the CSV download button
 default-active-fields : array, only show these columns at load time
+exclude-select-fields : array, column ids to hide from dropdown
```

Comment:
I feel like this needs to be more descriptive for newcomers to nova table. Perhaps in the description mention this excludes it from the active field list

Reply:
> 
> 
> I feel like this needs to be more descriptive for newcomers to nova table. Perhaps in the description mention this excludes it from the active field list

Good point, I've done that and taken it a step further and renamed the prop to a better descriptive term, alwaysActiveFields

Reply:
you have my +1 if others agree